docs/ref-manual/cli/options.md

```diff
@@ -399,6 +399,53 @@ Where `TokenPair` is
 We have two contracts `BankToken.sol` and `LoanToken.sol`. We want `tokenA` of the `tokenPair` to be `BankToken`, and `tokenB` to be `LoanToken`. Addresses take up only one slot. We assume `tokenPair` is the first field of Bank (so it starts at slot zero). To do that, we use:  
 `certoraRun Bank.sol BankToken.sol LoanToken.sol --verify Bank:Bank.spec --structLink Bank:0=BankToken Bank:1=LoanToken`
 
+Options for controlling contract creation
+-----------------------------------------
+
+### `--dynamic_bound`
+
+**What does it do?**
+If set to zero (the default) dynamic creation will not be modeled. If non-zero, then dynamic contract creation will be modeled with cloning, where each contract will be cloned at most n times.
+**When to use it?**
+When you wish to model contract creation as a non-havoc.
```

Comment:
I think that modelling does more than `non-havoc`, it simulates the actual creation of the contract as a full-fledged entity on our (fake) block chain. It may be worth mentioning the behavior *without* this turned on: "without it, the `create` and `create2` commands simply return a fresh address; the prover does not model their storage, code, constructors, immutables, etc. Any interaction with these generated addresses is modeled imprecisely with conservative havoc."